docs/source/support/support.rst

```diff
@@ -1,21 +1,36 @@
 .. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-.. _support-landing-page:
-
 Support and feedback
 ====================
 
 Have questions or feedback? You're in the right place.
 
-Questions: Stack Overflow
-	For “how do I?”, “why does something work this way” or “I’ve got a programming problem I’m trying to solve” questions, the `daml tag on Stack Overflow <https://stackoverflow.com/questions/tagged/daml>`_ is the best place to ask. 
+- **Questions: Stack Overflow**
+
+  For “how do I?”, “why does something work this way” or “I’ve got a programming problem I’m trying to solve” questions, the `daml tag on Stack Overflow <https://stackoverflow.com/questions/tagged/daml>`_ is the best place to ask. 
+
+  If you're not sure what makes a good question, take a look at `this checklist <https://codeblog.jonskeet.uk/2012/11/24/stack-overflow-question-checklist/>`_. 
+- **Help and feedback: Slack, or Support widget**
+
+  If you want to give feedback, or have questions that aren't right for Stack Overflow, you can: 
+
+  - Join the DAML community on `Slack <https://damldriven.slack.com/sso/saml/start>`_.
+  - Ask us for help using the `Support <javascript:open_feedback()>`_ link on any documentation page.
+
+When you're in the community Slack or on Stack Overflow, please keep to the `Code of Conduct <https://github.com/digital-asset/daml/blob/master/CODE_OF_CONDUCT.md>`__. 
+
+Support expectations
+--------------------
+
+For community users (ie on Slack and Stack Overflow):
```

Comment:
```suggestion
For community users (i.e. on Slack and StackOverflow):
```

Reply:
Actually, it is Stack Overflow! And following my usual style guide not to put the dots in ie and eg